Task 8/script.js

```diff
@@ -5,3 +5,23 @@ subtraction(a, b) - priima du skaičius ir grąžina jų skirtumą.
 multiplication(a, b) - priima du skaičius ir grąžina jų daugybos rezultatą;
 division(a, b) - priima du skaičius ir grąžina jų dalybos rezultatą;
 ------------------------------------------------------------------------------------ */
+
+function Calculator(a, b) {
+  this.a = a;
```

Comment:
No need to accept argument into this function, methods themselves should take those arguments.